src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -119,8 +119,12 @@ export default [
         icon: Expensicons.ChatBubble,
         successTextTranslateKey: '',
         successIcon: null,
-        shouldShow: (type, reportAction, isArchivedRoom, betas, anchor, isChronosReport, reportID) =>
-            type === CONTEXT_MENU_TYPES.REPORT_ACTION && reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && !ReportUtils.isThreadFirstChat(reportAction, reportID),
+        shouldShow: (type, reportAction, isArchivedRoom, betas, anchor, isChronosReport, reportID) => {
+            const isNormalAction = reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && !ReportUtils.isThreadFirstChat(reportAction, reportID);
+            const isPreviewAction = reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.REPORTPREVIEW;
+            const isIOUAction = reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU && !ReportActionUtils.isSplitBillAction(reportAction);
+            return type === CONTEXT_MENU_TYPES.REPORT_ACTION && (isNormalAction || isPreviewAction || isIOUAction);
```

Comment:
if (type !== CONTEXT_MENU_TYPES.REPORT_ACTION) return earlier

Reply:
isNormalAction -> isCommentAction
isPreviewAction -> isReportPreviewAction (as task preview is also preview)